MH4U Dex/RegionEncylcopediaCollectionViewCell.h

```diff
@@ -10,4 +10,12 @@
 
 @interface RegionEncylcopediaCollectionViewCell : UICollectionViewCell
 
+@property (nonatomic, strong) NSString *regionName;
+@property (nonatomic, strong) NSString *regionKeyName;
+
+/**
+ Update the cell's labels.
+ */
+- (void)displayContents;
```

Comment:
I could pass it a model object instead of storing the two strings as properties, if that would be preferable